Makefile

```
@@ -292,10 +254,11 @@ include make/protogen.mk
 .PHONY: go-easyjson-srcs
 go-easyjson-srcs: $(EASYJSON_BIN)
 	@echo "+ $@"
-	$(SILENT)easyjson -pkg pkg/docker/types/types.go
-	$(SILENT)easyjson -pkg pkg/docker/types/container.go
-	$(SILENT)easyjson -pkg pkg/docker/types/image.go
-	$(SILENT)easyjson -pkg pkg/compliance/compress/compress.go
+	@# Don't change the order of these, as otherwise the generation might become unstable.
```

I suggest to reword this comment to make it more actionable for future generations:
```suggestion
	@# Files are ordered such that repeated runs of `make go-easyjson-srcs` don't create diffs.
```
Or something like this.

---

Done